.travis.yml

```diff
@@ -1,15 +1,3 @@
 language: java
-jdk:
-  - oraclejdk8
-# A peculiarity of dependency caching in Gradle means that to avoid uploading the cache after every build the following
-# lines must be added:
-before_cache:
-  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
-  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
-cache:
-  directories:
-    - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
-install: true
```

Comment:
I don't remember what this was about, but it runs the command "/bin/true" as install script... probably a good thing it's removed 😃

Reply:
removed